purchase_requisition_from_rfq/models/purchase_requisition.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class PurchaseRequisition(models.Model):
+    _inherit = "purchase.requisition"
+
+    purchase_ids = fields.One2many(
+        default=lambda self: self._context.get('active_ids', []),
+    )
+    line_ids = fields.One2many(
+        default=lambda self: self._get_line(),
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form',
+                        toolbar=False, submenu=False):
+        res = super(PurchaseRequisition, self).fields_view_get(
+            view_id=view_id, view_type=view_type,
+            toolbar=toolbar, submenu=submenu)
+        if res and view_type in ['tree', 'form']:
+            orders = self.env['purchase.order'].browse(
+                self._context.get('active_ids', []))
+            order_lines = self.env['purchase.order.line']
+            te = orders.mapped('requisition_id')
+            if any(order.state not in ('draft', 'send', 'to approve')
```

Comment:
All selected order must be in 'draft' only? Why do we need 'send' and 'to approve' ?